File: 1009 - Salary with Bonus/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        scan.nextLine();
        double salary = scan.nextDouble();
        double sold_amount = scan.nextDouble();
        System.out.printf("TOTAL = R$ %.2f\n", salary + sold_amount * 0.15);
        scan.close();
    }
}

File: 1009 - Salary with Bonus/Main.c
#include <stdio.h>

int main()
{
    char name[10];
    gets(name);
    double salary, sold_amount;
    scanf("%lf %lf", &salary, &sold_amount);
    printf("TOTAL = R$ %0.2lf\n", salary + sold_amount * 0.15);
    return 0;
}